### ISSUE TITLE
Bitwarden Desktop starts with black screen (and not minimized).

### ISSUE THREAD

**Issue by Dominicus1165** [bitwarden/clients#2931]
### Steps To Reproduce

Start Bitwarden either manually or through autostart

### Option 1: (Auto)Start _without_ "start minimized to tray" enabled
- Program starts and is ready for action

### Option 2:(Auto)Start _with_ "start minimized to tray" enabled
- Program starts minimized and can be used by Bitwarden Addon for browser

### Actual Result

Bitwarden starts with black screen.\
It needs to be minimized to tray and reopened to work properly.

Normal minimize or normal close does not fix the problem.

### Screenshots or Videos

![Bitwarden Black Screen](https://user-images.githubusercontent.com/57621651/174476375-bee74d12-4519-4eef-bfe2-9bbfc8cd2e28.png)

### Additional Context

Removing and fresh install do not work.\

### Operating System

Windows

### Operating System Version

W11 Pro 21H2 Build 22000.739

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2022.5.1

**Comment by federicom09** [comment]
Hello @Dominicus1165, how are you?

We reviewed the issue and we are not experiencing it in the version 2022.6.1
Could you please update to the latest version and let us know if you are still facing the issue?

Thanks!

**Comment by natesubra** [comment]
I'm seeing the same issue on Win10 21H2 19044.1826
nvidia card+drivers 516.69

I can confirm that minimizing to the tray and restoring fixes the issue. Thanks for sharing that @Dominicus1165

**Comment by BloodyIron** [comment]
I'm seeing the same issue for many months (the whole time we've been using it?) on Ubuntu 20.04 and now 22.04. The window doesn't update to any content until I "open a new window" for Bitwarden, and clicking on the X symbol to close the window does nothing.

![image](https://user-images.githubusercontent.com/12439391/189187185-1b1fe9ca-8083-4a3b-99d3-ec846dbc180b.png)

**Comment by djsmith85** [comment]
@BloodyIron The version 1.25.1 is quite old (March 2021). Please update to a newer version and see if that fixes it. If not, please report back and add any additional information that might help us with reproducing this.

**Comment by Kyzoku** [comment]
I've been having the exact same issue described for months.  Different versions of Windows 11, problem persists on Bitwarden reinstall and still happens after a complete windows reinstall.  Multiple versions of Bitwarden used, at least 6 months of releases the problem has existed for me.

**Comment by BloodyIron** [comment]
@djsmith85 so yeah, I thought I was getting updates from a ppa that didn't exist (I had lost sight of this topic), so I've since updated, but I'm still get this issue.

**Comment by djsmith85** [comment]
> @djsmith85 so yeah, I thought I was getting updates from a ppa that didn't exist (I had lost sight of this topic), so I've since updated, but I'm still get this issue.

Thank you for updating and reporting back

**Comment by BloodyIron** [comment]
You're welcome! :)

**Comment by kssxs** [comment]
I'm experiencing this issue as well, on both my home PC and my work one.  Both are Windows 11 23H2, and I have the latest desktop version of Bitwarden installed (2023.10.1).  Bitwarden starts with Windows and opens to a black screen.  Does not start to tray and minimized the way I expected it to since that is what I set in the settings of the application.

**Comment by Greenderella** [comment]
Since this issue has been open for quite some time without recent activity, I'm curious if anyone is still experiencing this problem. I'm unable to reproduce this on Windows 11 Pro 23H2 + Bitwarden 2024.8

If the issue persists, please share any updated details. Thanks, everyone, for your contributions and feedback!

**Comment by Kyzoku** [comment]
I hadn't used the setting (Start to tray icon) in quite a while because of the black screen problem, so I activated it to test if this is still an issue and it was even worse than before.  Now it loaded up as a white screen and hung.  Closing the software and reopening refused to fix the issue and it made it impossible to revert the option.  In order for the software to work again I was forced to uninstall/reinstall the software.

**Comment by Kyzoku** [comment]
> I hadn't used the setting (Start to tray icon) in quite a while because of the black screen problem, so I activated it to test if this is still an issue and it was even worse than before. Now it loaded up as a white screen and hung. Closing the software and reopening refused to fix the issue and it made it impossible to revert the option. In order for the software to work again I was forced to uninstall/reinstall the software.

Edit: I tried it again while disabling Hardware Acceleration, but that didn't help.

**Comment by Dominicus1165** [comment]
Same. Only this very computer but even when reinstalling windows. 

@Kyzoku do you use OneDrive or any other similar solution? Maybe something gets transferred by that.

**Comment by Kyzoku** [comment]
I don't use any cloud storage.

**Comment by kssxs** [comment]
Yes this is still happening.  And as someone else mentioned, now it's a white screen instead of black.

![image](https://github.com/user-attachments/assets/1aab3a70-9391-4ae4-87a3-2d86d030781c)

**Comment by aussiedrongo** [comment]
I am also still experiencing this issue on the latest stable Bitwarden build running on a fully updated Windows 10 Home. While it's not a huge bother it is annoying to have to close it then reopen every time I restart.

**Comment by Aldnoa19** [comment]
Hello, i have the same problem. Running Windows 11 Pro for Workstations 23H2 22631.4317 
Windows updates are on, everything is updated.
In case it matters, i have nvidia rtx4070ti, 565.90 drivers and dual monitors.

**Comment by huntleyd** [comment]
Installing Bitwarden v2024.10.1 on Windows Pro 24H2 64-bit also resulted in a black screen for the app.  Could not open the window or maximize it (floating cursor over the Task Bar showed the black window).  After uninstallling, had the same experience attempting to install the current version from the Microsoft Store.

However, after uninstalling again I was able to install v2024.4.1 which I had saved from a previous install on a different computer.  This appeared to function normally.

I then looked at the version running normally on a *different* Win11 Pro computer (also 24H2), which was v2024.9.1.  I found the installer for this version and *without* uninstalling again on the problematic machine, went ahead and installed 'over the top' of the 2024.4.1 installation.  The 2024.9.1 version is now running normally on both Win11 Pro machines, although the app is prompting to update to the newest version - which I am resisting for now...

**Comment by pamposzek** [comment]
Using Bitwarden 2025.02 and Win 11 24H2 and experiencing same behavior. Nothing critical, but a bit annoying, having to click it to minimize every time on system startup. Otherwise, works normally.

**Comment by DevJoey** [comment]
I have the same issue on Android (A55)! Bitwarden starts with a blackscreen and hangs. After some Time my Phone says "Der Prozess Bitwarden reagiert nicht".

Its so annoying! I have to close, open, close, open bitwarden and so on to get it randomly to work. 

I already reported it via E-Mail the case is #600207

**Comment by swinzy** [comment]
On Fedora 42 and 43. Also having the same issue. Black window, can't close, have to kill the process and restart the app.

```
Version 2025.10.0
SDK 'main (d0262d3)'
Shell 36.9.3
Renderer 136.0.7103.177
Node 22.19.0
Architecture x64
```

**Comment by gabrielecabrini** [comment]
I also have the same issue with Gnome + Arch linux, the window is transparent

**Comment by dabeeeenster** [comment]
This is happening to me over and over on MacOS.

**Comment by kndenisenko** [comment]
Black Startup screen happens on Windows 11 and Fedora 42 Gnome (wayland). All OS has last updates

**Comment by trmartin4** [comment]
Please let us know if this is still occurring.  If not, we would like to close this issue, as it has been several months since last report, and we have had several Electron major version bumps since.

**Comment by huntleyd** [comment]
Well, given that I posted this in October 2024 (which at this point I don't even remember) and haven't experienced a black screen with Bitwarden in recent months, I think you can consider my issue resolved...!

Best regards,

David

________________________________

**Comment by pamperer562580892423** [comment]
@trmartin4 There may be another occurrence of this issue: https://github.com/bitwarden/clients/issues/20978

**Comment by lsm5** [comment]
I use the bitwarden desktop client installed via flatpak . I don't see a "start minimized" option. IIRC, it used to be there until about two months ago (or it somehow started minimized back then), but that hasn't been the case recently. I can file a separate issue for it if need be.

**Comment by pamperer562580892423** [comment]
> I can file a separate issue for it if need be.

@lsm5 I don't think that will be successful as that is an intentional change and not a bug - see the latest [Release Notes 2026.7.0](https://bitwarden.com/help/releasenotes/#2026-7-0):

<img width="771" height="139" alt="Image" src="https://github.com/user-attachments/assets/bee0e867-3d76-47c4-b248-f18943c2fa72" />
